<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Load_Arch</class>
 <widget class="QWidget" name="Load_Arch">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>564</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(238, 238, 238, 255), stop:1 rgba(190, 190, 190, 255));
font-family: Roboto;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="main">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(50, 98, 115, 40);
font-size: 11pt;
border: 1px solid rgba(50, 98, 115, 255);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="5" column="2" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLabel" name="label_status">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(50, 98, 115, 100);
color: #13242B;
border: 1px solid #326273;
font-weight: bold;
font-size: 10pt;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="2">
       <widget class="QPushButton" name="pushButton_yes">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(50, 98, 115, 190);
font-size: 14pt;
color: White;
border: None;
}

QPushButton::hover {
background-color: rgba(50, 98, 115, 255);
border: 2px solid rgba(92, 158, 173, 255);
}

QPushButton::pressed {
background-color: rgba(92, 158, 173, 255);
border: 1px solid rgba(255, 255, 255, 255);
}

QPushButton::disabled {
background-color: #EEEEEE;
border: 1px solid rgba(50, 98, 115, 255);
}

</string>
        </property>
        <property name="text">
         <string>Да, создать новую
историю болезни на
основе архивной</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
	color: #326273;
	border: none;
	background-color: none;
}

QCheckBox:checked {
   color: #702632;
}

QCheckBox:disabled {
   color: #EEEEEE;
}
 

QCheckBox::indicator:checked {
	border-style:solid;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: #702632;
}
QCheckBox::indicator:!checked {
	border-style:solid;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: none;
}

QCheckBox::indicator:disabled {
	border-style:solid;
	border-width: 1px;
	border-color: #EEEEEE;
	color: #FFFFFF;
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string>Подтверждаю отказ
от копирования</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #326273;
font-weight: bold;
background-color: transparent;
border: None;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Обнаружены пациенты в архиве историй болезней, &lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;с идентичными ФИО и датой рождения&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="4">
       <widget class="QFrame" name="copy_frame">
        <property name="styleSheet">
         <string notr="true">border: none;
background-color: transparent;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #326273;
font-weight: bold;
background-color: transparent;
border: None;</string>
           </property>
           <property name="text">
            <string>Режим копирования</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fullcopy">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: #326273;
	border: none;
	background-color: none;
}
QRadioButton:checked {
   color: #702632;
}
QRadioButton::indicator:checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: #702632;
}
QRadioButton::indicator:!checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string>полное</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="adm_copy">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: #326273;
	border: none;
	background-color: none;
}
QRadioButton:checked {
   color: #702632;
}
QRadioButton::indicator:checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: #702632;
}
QRadioButton::indicator:!checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string>только данные при поступлении</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dis_to_adm_copy">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: #326273;
	border: none;
	background-color: none;
}
QRadioButton:checked {
   color: #702632;
}
QRadioButton::indicator:checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: #702632;
}
QRadioButton::indicator:!checked {
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #702632;
	color: #FFFFFF;
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string>вставить данные при выписке
как данные при поступлении</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTableWidget" name="tableWidget">
        <property name="styleSheet">
         <string notr="true">QTableWidget {
   selection-background-color: rgba(50, 98, 115, 120);
   background-color: rgba(50, 98, 115, 40);
   color: #FFFFFF;
   font-weight: Normal;
   border-color: rgba(50, 98, 115, 255);
}

QHeaderView::section {
   background-color: rgba(50, 98, 115, 220);
   color: #FFFFFF;
   font-family: Roboto;
   font-size: 15px;
   font-weight: Normal;
}

QTableWidget::item {
   selection-background-color: rgba(50, 98, 115, 120);
   background-color: rgba(50, 98, 115, 40);
   color: #13242B;
   font-size: 15px;
   font-weight: Normal;
   border-style: none;
   border-bottom: #13242B;
}

QScrollBar {
background-color: rgba(50, 98, 115, 70);
}
</string>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ФИО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>МКБ-10</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Даты</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>uin</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: #326273;
font-weight: bold;
background-color: transparent;
border: None;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Использовать данные архива как&lt;br/&gt;основу новой истории болезни?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_no">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgba(50, 98, 115, 190);
font-size: 14pt;
color: White;
border: None;
}

QPushButton::hover {
background-color: rgba(50, 98, 115, 255);
border: 2px solid rgba(92, 158, 173, 255);
}

QPushButton::pressed {
background-color: rgba(92, 158, 173, 255);
border: 1px solid rgba(255, 255, 255, 255);
}

QPushButton::disabled {
background-color: #EEEEEE;
border: 1px solid rgba(50, 98, 115, 255);
}

</string>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
